packages/itwinui-css/src/radio-tile/radio-tile.scss

```diff
@@ -3,6 +3,12 @@
 @use '../mixins';
 
 .iui-radio-tile {
+  --_iui-radio-tile-color-icon: var(--_iui-radio-tile-color-icon-default);
+  --_iui-radio-tile-color-icon-default: var(--iui-color-icon-muted);
+  --_iui-radio-tile-color-icon-hover: var(--iui-color-icon-hover);
+  --_iui-radio-tile-color-icon-checked: var(--iui-color-icon-accent);
+  --_iui-radio-tile-color-icon-disabled: var(--iui-color-icon-disabled);
```

Comment:
Those details are still too vague. I get the general idea, but I'm looking for specifics of each new variable, otherwise there is no way to know why some of these variables were added.

Reply:
This explanation is unhelpful tbh. Like I said, I get the general idea, so I was looking for specifics on what each new variable is doing.

For example, I painstakingly figured out that in #2373, ProgressLinear/ProgressRadial is changing the visual style to use neutral colors (Kiwi) instead of accent ones (iTwinUI v3).

In the case of RadioTile, I'm not sure that these variables are adding any value. The bridge values from #2373 look similar to what we are setting in iTwinUI v3 in this PR.

https://github.com/iTwin/iTwinUI/blob/864fbe21216737ebc30fab7227a2e5e8d9fc9b00/packages/itwinui-css/src/bridge.scss#L269-L277

Every single component-level override in #2373 needs to have a solid explanation, otherwise it would be very hard to maintain.